less/layout.less

```diff
@@ -5,4 +5,39 @@ html, body {
 h1, h2, h3, h4, h5 {
     font-family: 'Russo One', sans-serif;
     color: @secondary-color;
-}
\ No newline at end of file
+}
+
+
+.header {
+    display: flex;
+    justify-content: center;
+    background-color: @accent-color;
+}
+
+
+ nav {
+     display: flex;
+     justify-content: space-around;
+ }
+
+ .main-content{
+     padding: 20px;
+ }
+
+ .images {
+     padding: 20px;
+     margin-right: 30px;
+ }
+
+ .visit-info {
+     padding: 20px;
+ }
+
+ .walk {
+     padding: 20px;
+ }
+
+ footer {
```

Comment:
This footer is a little "smushed" more margin and padding spacing would be good.